database/migrations/2020_09_26_124457_add_facebook_login.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddFacebookLogin extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create(
+            "social_profiles",
+            function (Blueprint $table): void {
+                $table->uuid("user_id")->primary();
+                $table->foreign("user_id")->references("id")->on("users");
+                $table->string("facebook_id")->nullable();
```

Comment:
@MLencki, what do you think? To be honest, I was thinking more about `id | user_id | provider(_name/_id) | id` approach. It would be easier to add new providers in the future.

Reply:
Yep, `provider_id` sounds more flexible. Also the name of the table `social_profiles` suggests that it can store profiles from different providers, not only `facebook_profiles`.

Reply:
Should it be `nullable` then?

Reply:
Probably not. Laravel\Socialite\Contracts\User implements 'getId()' method and provider_id is mandatory to compare returned 3rd party profile ID with one saved in our database.